### ISSUE TITLE
[BUG]: conflicting playbooks

### ISSUE THREAD

**Issue by S3nsh1r0** [meetrevision/playbook#120]
### Describe the bug

Whilst trying to upgrade to Revi 24.06 on top of Windows 22H2 19045, I get the following error:
```
Conflicting Playbooks
This playbook is already applied to your system and doesn't support this upgrade
v23.12 > v24.06
```

### Steps to reproduce

1. install W10 22h2 19045
2. Install Revi playbook v23.12
3. try installing v.24.06
4. observe error

### Expected behavior

I expected the upgrade to be possible

### Environment

- OS Build Number & Edition: `Win10 Pro 22h2, 19045.4412,`
- ReviOS Version: `v23.12`
- Specs: `CPU: Intel(R) Core(TM) i5-7300U (4) @ 3,50 GHz; GPU: Intel(R) HD Graphics 620 (128,00 MiB) [Integrated]; Host: 20HGS4BW00 (ThinkPad T470s)`

### Additional information

This also occurs on another device running the same setup.

### Have you searched for similar issues?

- [X] I have searched for similar issues and found none.

**Comment by irkella** [comment]
I have done this upgrade on the same version of Windows, so that sounds like it's down to your individual setup.

**Comment by Pyenb** [comment]
You can just click continue. That behavior is expected 👍

**Comment by melo936** [comment]
24.06 was released before [AME Wizard 0.7.5](https://docs.ameliorated.io/changelog/2024-06-12.html) and we weren't aware of the new changes. It will be resolved in next updates.

For now, like @Pyenb said, you can just click the Continue button.

**Comment by S3nsh1r0** [comment]
Ah okay, so my system won´t have broken software afterwards?
